### PR TITLE
Final/tree validation 

### PR DESCRIPTION
**Pull request by skymrsn**

### CODE REVIEW

File: iOS/Podfile [hunk 1]
```diff
@@ -21,3 +21,11 @@ target 'iSeaTree' do
...
 
 end
+post_install do |installer_representation|
+  installer_representation.pods_project.targets.each do |target|
+      target.build_configurations.each do |config|
+          config.build_settings['ONLY_ACTIVE_ARCH'] = 'NO'
+          # config.build_settings['BUILD_LIBRARY_FOR_DISTRIBUTION'] = 'NO'
+      end
+  end
+end
```
Comment (mtalhadev) [inline iOS/Podfile:31]: Please just keep it locally, don't push it for now. As right now at this point, we don't want to put this permanently.

File: iOS/Podfile.lock [hunk 2]
```diff
@@ -378,25 +378,25 @@ PODS:
...
-  - GoogleUtilities/ISASwizzler (7.5.2)
-  - GoogleUtilities/Logger (7.5.2):
+  - GoogleUtilities/ISASwizzler (7.7.0)
+  - GoogleUtilities/Logger (7.7.0):
     - GoogleUtilities/Environment
-  - GoogleUtilities/MethodSwizzler (7.5.2):
+  - GoogleUtilities/MethodSwizzler (7.7.0):
     - GoogleUtilities/Logger
-  - GoogleUtilities/Network (7.5.2):
+  - GoogleUtilities/Network (7.7.0):
```
Comment (mtalhadev) [inline iOS/Podfile.lock:392]: Please don't push .lock file unless there's a new library.

**Comment by mtalhadev** [comment]
Hi @skymrsn , can you remove everything  except these
-src folder
-react-native-modal from node_modules
-package.json

**Comment by skymrsn** [comment]
> Hi @skymrsn , can you remove everything except these -src folder -react-native-modal from node_modules -package.json

File: iOS/Podfile.lock (hunk 2 above)
Comment (skymrsn) [reply iOS/Podfile.lock:392]: how am I able to only push those three files you mentioned? Is it .gitignore or certain push command?

**Pull request closed without merging**